centos2ol_wrapper.sh

```diff
@@ -0,0 +1,407 @@
+#!/bin/bash
+# centos2ol_wrapper -- Wrapper with sanity checks for centos2ol
+# Copyright Nikolai Bezroukov, 2020. Licensed under Perl Artistic license.
+#:: 
+#:: The script accepts options -d, -b -p, -r (other than -h which print this help screen and -v for verbose output)
+#:: You can also adapt configuration parameters that set defaults for those options 

```

Comment:
All of the functionality of this script should be included/incorporated into `centos2ol.sh`.